Typstify/Typstify/ContentView.swift

```diff
@@ -359,6 +360,8 @@ struct Navigator: View {
                                                     $to: viewState.dominantFolder!,
                                                     withPreferredName: "\(name)"
                                                 )
+                                                try data.write(to: URL(fileURLWithPath: fullPath))
```

Comment:
It seems we needed to add code to write image data to the system.

Reply:
Have you tested this on your iPad? It seemed to me that this will actually be in conflict with Swift's file management saving. That's why I didn't take this approach.

Please attach a screen recording of this working if possible.